language/stdlib/src/transaction_scripts.rs

```diff
@@ -55,6 +55,7 @@ pub enum StdlibScript {
     RotateConsensusPubkey,
     RotateSharedEd2551PublicKey,
     TieredMint,
+    TravelRuleLimitUpdate,
```

Comment:
Nit: verb first (`UpdateTravelRuleLimit`) is the pattern used  in most of  these.